crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -469,6 +469,10 @@ define_tedge_config! {
             /// Set of SmartREST template IDs the device should subscribe to
             #[tedge_config(example = "templateId1,templateId2", default(function = "TemplatesSet::default"))]
             templates: TemplatesSet,
+
+            /// Switch using 501-503 (without operation ID) or 504-506 (with operation ID) SmartREST messages for operation status update
+            #[tedge_config(example = "true", default(value = true))]
+            with_operation_id: bool,
```

Comment:
```suggestion
            direct_operation_update: bool,
```
Not fully happy about this suggestion either. Accept only if you're convinced.

Reply:
The key will look `c8y.smartrest.direct_operation_update`. I don't think what "direct" means is so clear. At least, not for me. How about something in between my proposal and yours, `c8y.smartrest.operation_id_update`?

Reply:
My bikeshed take would be something like `c8y.smartrest.operation_explicit_id` or `operation_template_explicit_id`

Reply:
I see no reason to change anything here: `with_operation_id` is simple and clear.

Reply:
I vote +1 for `use_operation_id`